build-system/eslint-rules/html-template.js

```diff
@@ -177,7 +177,7 @@ function create(context) {
     // Void tags are defined at
     // https://html.spec.whatwg.org/multipage/syntax.html#void-elements
     const invalid =
-      /<(?!area|base|br|col|embed|hr|img|input|link|meta|param|source|track|wbr)([a-zA-Z-]+)([\s\n][^>]*)?\/>/gm;
+      /<(?!area|base|br|col|embed|hr|img|input|link|meta|param|source|track|wbr)(((svg)?)([a-zA-Z-])+)([\s\n][^>]*)?\/>/gm;
```

Comment:
It seems `svgSelfClosing` and related code can be cleaned up.

Reply:
This could mean a lot of things. Can you clarify what you mean by 'cleaning up'?

Reply:
The link you provided says that all foreign elements are valid void tags, so the automatic "fix" isn't necessary at all for SVG elements. Can you delete everything related to that?

Reply:
https://html.spec.whatwg.org/multipage/syntax.html#void-elements

If you scroll up just a tad

"There are six different kinds of elements: void elements, the template element, raw text elements, escapable raw text elements, foreign elements, and normal elements."

svg is considered a foreign element, so I don't think we should consider them as void tags.

We have a few options here, but one solution could be:

make invalidVoidTag => invalidTag, and add svg to the list of tags

I think a quick meeting can help us figure this out faster - i've set a few minutes for us to meet tomorrow.

Reply:
See https://html.spec.whatwg.org/multipage/syntax.html#syntax-start-tag, Step 6

> Then, if the element is one of the void elements, or if the element is a foreign element, then there may be a single U+002F SOLIDUS character (/). This character has no effect on void elements, but on foreign elements it marks the start tag as self-closing.

Reply:
@jridgewell just cleaned-up and ran some tests, so we should be good to go -
 give it a look over when you get the chance and let me know if you need anything else.